packages/core/src/browser/messaging/ws-connection-provider.ts

```diff
@@ -68,6 +68,24 @@ export class WebSocketConnectionProvider extends AbstractConnectionProvider<WebS
         }
     }
 
+    protected doOpenChannel(path: string, handler: (channel: WebSocketChannel) => void, options?: WebSocketOptions): void {
```

Comment:
I'm sorry but I don't understand that change. How is it different from `super.openChannel(...)`?

Reply:
You're right! Thanks a lot 🙏